docs/cody/core-concepts/context.mdx

```diff
@@ -17,10 +17,9 @@ Context and [methods of retrieving context](#context-selection) are crucial to t
 
 Cody employs various methods to gather context relevant to user input, ensuring the quality of the information provided. These methods include:
 
-- **Local**: Sparse vector search running on a user's machine. Creates a locally computed and stored search index of an active repo. Queries are converted to an expanded set of keywords, which are used to retrieve relevant documents from the index
-- **Cody Context Search**: Sourcegraph Search API. Queries are sent to the SG instance (managed or self-hosted), and search is done using the SG search stack. Relevant documents are returned to the user IDE for use by the LLM
-- **Embeddings**: Dense vector search requiring both documents (code) and queries (chats) to be converted to an embedding representation via an embeddings model. Retrieval is then done by comparing the relationship between the query and document embedding vectors
-- **Keyword Search**: A traditional text search approach that finds keywords matching user input in local code. This method is used when a codebase lacks embeddings
+- **Keyword Search**: A traditional text search approach that finds keywords matching the user input. When needed, queries are automatically rewritten to include more relevant terms.
```

Comment:
I snuck in some fixes here too, although they are not strictly related to context limits:
* "Local" and "Keyword Search" are actually the same thing
* It's clearer to call it "Sourcegraph Search" instead of "Cody Context Search"

Reply:
These look good. :)